backend/src/services/opportunity.service.ts

```diff
@@ -0,0 +1,71 @@
+import { FindManyOptions, getCustomRepository } from 'typeorm';
+import OpportunitiesRepository from '~/data/repositories/opportunity.repository';
+import CompanyRepository from '~/data/repositories/company.repository';
+
+import { Company } from '~/data/entities/company';
+import { Opportunity } from '~/data/entities/opportunity';
+
+import { opportunityMapper } from '~/common/mappers/opportunities.mapper';
+import UserRepository from '~/data/repositories/user.repository';
+import { User } from '~/data/entities/user';
+import { users } from '~/data/seed-data/user.data';
+import { asyncForEach } from '~/common/helpers/array.helper';
+
+export const getOpportunities = async (
+  companyId: string,
+): Promise<Opportunity[]> => {
+  const opportunityRepository = getCustomRepository(OpportunitiesRepository);
+  const companyRepository = getCustomRepository(CompanyRepository);
+  const userRepository = getCustomRepository(UserRepository);
+  const companyInstance: Company = await companyRepository.findOne({
+    id: companyId,
+  });
+  const userInstance: User = await userRepository.findOne({
+    firstName: users[0].firstName,
+  });
+  const opportunities = await opportunityRepository.find({
+    company: companyInstance,
+    user: userInstance,
+    relations: ['company', 'user', 'tags'],
+  } as FindManyOptions);
```

Comment:
As I understood, this service should return all opportunities in the company. You do not need user here